docs/source/federation-version-support.mdx

```diff
@@ -5,220 +5,55 @@ title: Federation version support in the Apollo Router
 import { Link } from 'gatsby';
 
 The Apollo Router uses Apollo Federation 2.x. Federation 2.x
-is [backward compatible with Federation 1.x](https://www.apollographql.com/docs/federation/federation-2/backward-compatibility/), so you can use the router with a valid Federation 1.x supergraph.
+is [backward compatible with Federation 1.x](/federation/federation-2/backward-compatibility/), so you can use the router with a valid Federation 1.x supergraph.
 
 > If your Federation 1.x supergraph _doesn't_ work with the Apollo Router, see [Backward compatibility in Federation 2](/federation/federation-2/backward-compatibility/) for possible causes.
 
 Apollo Federation is an evolving project, and it will receive new features and bug fixes over time. If you need to run a particular version of federation, the following table shows which version of federation each router release is compiled against:
 
-<table>
+<table class="field-table">
```

Comment:
It seems easier and more digestible for this table to include rows of version _ranges_, instead of noting the federation version of each individual version